ec2-asg-dynamic-target-tracking/message-producer/app.py

```diff
@@ -0,0 +1,59 @@
+
+# Import modules
+import boto3
+import os
+import random
+import os 
+import struct
+import json
+from pprint import pprint
+import logging
+from botocore.exceptions import ClientError
+from botocore.config import Config
+
+# Create logger
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
+
+# Define config
+config = Config(
+   retries = {
+      'max_attempts': 10,
+      'mode': 'standard'
+   }
+)
+
+# Define session and resources
+session = boto3.Session()
+sqs = session.resource('sqs', config=config)
+# cloudwatch = session.client('cloudwatch', config=config)
+# autoscaling = session.client('autoscaling', config=config)
+
+# Read  environment variables
+queueName = os.environ['queueName']
+defaultMsgProcDuration = int(os.environ['defaultMsgProcDuration'])
+
+# Initialize other parameters
+nMessages = 50
+
+def lambda_handler(event, context):
+
+    # Get the queue
+    queue = sqs.get_queue_by_name(QueueName=queueName)
+
+    # Send N messages
+    for i in range(nMessages): 
+        
+        # Build Msg body
+        randomNumber = struct.unpack('H', os.urandom(2))[0]
+        messageBody = {"id": randomNumber, "duration": defaultMsgProcDuration}
+        print('Sending message id: {}'.format(messageBody.get('id')))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `get` method without a default argument to return the value of a key in a dictionary. We recommended that you use a default argument so that if the value for your key is not found, a default value is returned. If a default value is not provided and the key is not found, then `None` is returned.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)